pipelines/broad/arrays/single_sample/Arrays.wdl

```diff
@@ -504,14 +504,14 @@ workflow Arrays {
     File? output_fingerprint_vcf = IlluminaGenotypingArray.output_fingerprint_vcf
     File? output_fingerprint_vcf_index = IlluminaGenotypingArray.output_fingerprint_vcf_index
     File? output_fingerprint_json_file = VcfToMercuryFingerprintJson.output_json_file
-    File arrays_variant_calling_detail_metrics_file = select_first([IlluminaGenotypingArray.arrays_variant_calling_detail_metrics, GenerateEmptyVariantCallingMetricsFile.detail_metrics])
     File? arrays_variant_calling_summary_metrics_file = IlluminaGenotypingArray.arrays_variant_calling_summary_metrics
+    File arrays_variant_calling_detail_metrics_file = select_first([IlluminaGenotypingArray.arrays_variant_calling_detail_metrics, GenerateEmptyVariantCallingMetricsFile.detail_metrics])
     File? arrays_variant_calling_control_metrics_file = IlluminaGenotypingArray.arrays_variant_calling_control_metrics
-    File? arrays_subset_variant_calling_detail_metrics_file = IlluminaGenotypingArray.arrays_subset_variant_calling_detail_metrics
     File? arrays_subset_variant_calling_summary_metrics_file = IlluminaGenotypingArray.arrays_subset_variant_calling_summary_metrics
+    File? arrays_subset_variant_calling_detail_metrics_file = IlluminaGenotypingArray.arrays_subset_variant_calling_detail_metrics
```

Comment:
@gbggrant We'll need to update the changelog and version number again to reflect this change.

Reply:
Not worth it.  I'll just put them back in the order they were originally (this was a very trivial, non-function change).